GPU intermittently resets on Linux with AMD graphics card

### Summary

<!-- Begin your issue with a one sentence summary -->
Zed somehow causes gpu reset

### Description
<!-- Include all steps necessary to reproduce from a clean Zed installation. Be verbose -->
Steps to trigger the problem:
1. Just use zed
2. It randomly crashes
3. Dmesg log https://imgur.com/a/asN5gCg

Actual Behavior: crash
Expected Behavior: stable work


### Zed Version and System Specs

Zed: v0.181.6 (Zed) 
OS: Linux Wayland arch unknown
Memory: 31.1 GiB
Architecture: x86_64
GPU: AMD Radeon RX 7600 XT (RADV NAVI33) || radv || Mesa 25.0.3-arch1.1

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.

<details><summary>Zed.log</summary>

<!-- Paste your log inside the code block. -->
```log
2025-04-16T12:42:24.549512558+03:00 [ERROR] {
  "thread": "main",
  "payload": "GPU has crashed, and no debug information is available.",
  "location_data": {
    "file": "/build/.cargo/git/checkouts/blade-26bd638d88226de3/b16f5c7/blade-graphics/src/vulkan/command.rs",
    "line": 448
  },
  "backtrace": [
    "__libc_start_main+30628273370444"
  ],
  "app_version": "0.181.6",
  "release_channel": "stable",
  "target": "x86_64-unknown-linux-gnu",
  "os_name": "Linux Wayland",
  "os_version": "arch unknown",
  "architecture": "x86_64",
  "panicked_on": 1744796544549,
  "system_id": "c3b901a4-3bcc-4486-bd3c-11134c3de3d7",
  "installation_id": "68f06f9c-79d0-4f49-b3dd-3fbf881120f4",
  "session_id": "1f4366d6-3aa2-4549-a793-1ff84b077f1e"
}
```

</details>

--- a/docs/src/linux.md
+++ b/docs/src/linux.md
@@ -151,7 +151,7 @@ If you're using an AMD GPU and Zed crashes when selecting long lines, try settin
 
 If you're using an AMD GPU, you might get a 'Broken Pipe' error. Try using the RADV or Mesa drivers. (See [#13880](https://github.com/zed-industries/zed/issues/13880))
 
-If you are using `amdvlk` you may find that zed only opens when run with `sudo $(which zed)`. To fix this, remove the `amdvlk` and `lib32-amdvlk` packages and install mesa/vulkan instead. ([#14141](https://github.com/zed-industries/zed/issues/14141)).
+If you are using `amdvlk`, the default open-source AMD graphics driver, you may find that Zed consistently fails to launch. This is a known issue for some users, for example on Omarchy (see issue [#28851](https://github.com/zed-industries/zed/issues/28851)). To fix this, you will need to use a different driver. We recommend removing the `amdvlk` and `lib32-amdvlk` packages and installing `vulkan-radeon` instead (see issue [#14141](https://github.com/zed-industries/zed/issues/14141)).
 
 For more information, the [Arch guide to Vulkan](https://wiki.archlinux.org/title/Vulkan) has some good steps that translate well to most distributions.
 

